packages/fx-core/src/common/deps-checker/internal/funcToolChecker.ts

```diff
@@ -24,6 +25,26 @@ export enum FuncVersion {
   v4 = "4",
 }
 
+const FuncNodeVersionWhiteList: { [key: string]: { [key: string]: boolean } } = {
```

Comment:
nit: You can declare like this to avoid `!` in `FuncNodeVersionWhiteList[funcVersion.toString()]![nodeVersion]`
```suggestion
const FuncNodeVersionWhiteList: { [key: FuncVersion]: { [key: string]: boolean } } = {
```

Reply:
![image](https://user-images.githubusercontent.com/12523371/170189416-170a7501-8f2b-4a5e-8bfb-25775fbdd942.png)
anyway, the object of func version may be empty that need `!`

Reply:
![image](https://user-images.githubusercontent.com/9698542/170190438-07b6fef8-81f8-48d4-a2da-0db3bc3af24c.png)